fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -129,164 +135,167 @@
-    private boolean enableBucketShuffleJoin = false;
+    public boolean enableBucketShuffleJoin = false;
 
     @VariableMgr.VarAttr(name = PREFER_JOIN_METHOD)
-    private String preferJoinMethod = "broadcast";
+    public String preferJoinMethod = "broadcast";
 
     /*
      * the parallel exec instance num for one Fragment in one BE
      * 1 means disable this feature
      */
     @VariableMgr.VarAttr(name = PARALLEL_FRAGMENT_EXEC_INSTANCE_NUM)
-    private int parallelExecInstanceNum = 1;
+    public int parallelExecInstanceNum = 1;
 
-    @VariableMgr.VarAttr(name = ENABLE_INSERT_STRICT)
-    private boolean enableInsertStrict = false;
+    @VariableMgr.VarAttr(name = ENABLE_INSERT_STRICT, needForward = true)
+    public boolean enableInsertStrict = false;
 
     @VariableMgr.VarAttr(name = ENABLE_ODBC_TRANSCATION)
-    private boolean enableOdbcTransaction = false;
+    public boolean enableOdbcTransaction = false;
 
     @VariableMgr.VarAttr(name = ENABLE_SQL_CACHE)
-    private boolean enableSqlCache = false;
+    public boolean enableSqlCache = false;
 
     @VariableMgr.VarAttr(name = ENABLE_PARTITION_CACHE)
-    private boolean enablePartitionCache = false;
+    public boolean enablePartitionCache = false;
 
     @VariableMgr.VarAttr(name = FORWARD_TO_MASTER)
-    private boolean forwardToMaster = false;
+    public boolean forwardToMaster = false;
 
     @VariableMgr.VarAttr(name = LOAD_MEM_LIMIT)
-    private long loadMemLimit = 0L;
+    public long loadMemLimit = 0L;
 
     @VariableMgr.VarAttr(name = USE_V2_ROLLUP)
-    private boolean useV2Rollup = false;
+    public boolean useV2Rollup = false;
 
     // TODO(ml): remove it after test
     @VariableMgr.VarAttr(name = TEST_MATERIALIZED_VIEW)
-    private boolean testMaterializedView = false;
+    public boolean testMaterializedView = false;
 
     @VariableMgr.VarAttr(name = REWRITE_COUNT_DISTINCT_TO_BITMAP_HLL)
-    private boolean rewriteCountDistinct = true;
+    public boolean rewriteCountDistinct = true;
 
     // compatible with some mysql client connect, say DataGrip of JetBrains
     @VariableMgr.VarAttr(name = EVENT_SCHEDULER)
-    private String eventScheduler = "OFF";
+    public String eventScheduler = "OFF";
     @VariableMgr.VarAttr(name = STORAGE_ENGINE)
-    private String storageEngine = "olap";
+    public String storageEngine = "olap";
     @VariableMgr.VarAttr(name = DIV_PRECISION_INCREMENT)
-    private int divPrecisionIncrement = 4;
+    public int divPrecisionIncrement = 4;
 
     // -1 means unset, BE will use its config value
     @VariableMgr.VarAttr(name = MAX_SCAN_KEY_NUM)
-    private int maxScanKeyNum = -1;
+    public int maxScanKeyNum = -1;
     @VariableMgr.VarAttr(name = MAX_PUSHDOWN_CONDITIONS_PER_COLUMN)
-    private int maxPushdownConditionsPerColumn = -1;
+    public int maxPushdownConditionsPerColumn = -1;
     @VariableMgr.VarAttr(name = SHOW_HIDDEN_COLUMNS, flag = VariableMgr.SESSION_ONLY)
```

Comment:
Why change these session variables to public?

Reply:
You can sue method `setForwardedSessionVariables()` in SessionVariables.java.
I use ` SessionVariable.class.getFields();` to find field, and it can only return public field.